git-cliff-core/src/commit.rs

```diff
@@ -728,4 +727,55 @@ mod test {
 
 		Ok(())
 	}
+
+	#[test]
+	fn field_name_regex() -> Result<()> {
```

Comment:
Thanks for adding the unit test! Can you also [add a fixture](https://github.com/orhun/git-cliff/tree/main/.github/fixtures) for testing the behavior?